pkg/minishift/openshift/oc.go

```diff
@@ -42,3 +46,24 @@ func AddSudoersRoleForUser(user string) error {
 	}
 	return nil
 }
+
+// Add Current Profile Context
+func AddContextForProfile(profile string, ip string) error {
+	cmdName := instanceState.Config.OcPath
+	ip = strings.Replace(ip, ".", "-", -1)
+	cmdArgs := []string{"config", "set-context", profile,
+		fmt.Sprintf("--cluster=%s:%d", ip, constants.APIServerPort),
+		fmt.Sprintf("--user=developer/%s:%d", ip, constants.APIServerPort),
+		"--namespace=myproject"}
```

Comment:
I am wondering the same thing. Is the _namespace_ really required? Can it not be omitted?